stories/features/rewards/concepts.tsx

```diff
@@ -100,6 +104,10 @@ storiesOf('Feature Components/Rewards/Concepts/Desktop', module)
       return screenName.length > 0 || tweetText.length > 0
     }
 
+    const isRedditTip = () => {
+      return !isTwitterTip() && (redditUserName.length !== 0 && redditPost.length !== 0)
```

Comment:
do we need this check `!isTwitterTip()` as `isRedditTip()` is else to twitter check

Reply:
No, it is not needed